src/sentry/static/sentry/app/views/releases/organizationReleases/index.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import styled from 'react-emotion';
+import PropTypes from 'prop-types';
+
+import SentryTypes from 'app/sentryTypes';
+import {t} from 'app/locale';
+
+// import SearchBar from 'app/components/searchBar';
+import {Panel, PanelBody} from 'app/components/panels';
+import Pagination from 'app/components/pagination';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import Alert from 'app/components/alert';
+
+import withOrganization from 'app/utils/withOrganization';
+import withGlobalSelection from 'app/utils/withGlobalSelection';
+
+import space from 'app/styles/space';
+
+import ReleaseList from '../shared/releaseList';
+import ReleaseListHeader from '../shared/releaseListHeader';
+import {fetchOrganizationReleases} from '../shared/utils';
+
+class OrganizationReleases extends React.Component {
```

Comment:
Should this be an AsyncView?

Reply:
It could be right now, but I suspect that will make things more complicated when we add the project/env/time/querystring filtering to this. From what I've seen asynccomponent is a better fit when component loads data once on mount.